SetupDataPkg/Tools/SettingSupport/UefiVariablesSupportLinuxLib.py

```diff
@@ -0,0 +1,140 @@
+# @file
+#
+# Python lib to support Reading and writing UEFI variables from Linux
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c), Microsoft Corporation
+# SPDX-License-Identifier: MIT

```

Comment:
Shouldn't this still be BSD-Clause-Patent?

Reply:
modified in latest push based on off line conversations.